### DISCUSSION TITLE
[SF][Select] Value state message UI is unexpected.

### DISCUSSION THREAD

**Discussion by ChristyFei** [UI5/webcomponents#10300] [Q&A]
Hi Colleagues,

The customized value state message does not display as expected, but the default value state message works fine.
![Screenshot 2024-12-04 at 11 24 59](https://github.com/user-attachments/assets/783b5b7a-dbae-43fa-9854-d9e2abc10e1c)

Default Value State Message:
![Screenshot 2024-12-04 at 11 25 15](https://github.com/user-attachments/assets/399b3c68-f89b-4652-b15a-2e9e621da9de)

**Reproduce Steps:**
1. Open Sample https://codesandbox.io/p/sandbox/exciting-snow-39hkwn?file=%2Fsrc%2FApp.js%3A3%2C58 
2. Click on Select. The dropdown list and the error message are shown.
3. Click again. The dropdown list is closed. The error message popup is still shown without messages. 

**Expected Behavior:**
The customized error message behavior should be consistent with the default error message. 

Thank you,
Christy

**Comment by nnaydenow** [comment]
Hi @ChristyFei,

This is an issue with the component and I created an issue from this discussion where you can track its progress.
https://github.com/SAP/ui5-webcomponents/issues/10323

Regards,
Nayden
